packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -35,20 +35,52 @@ export enum SiteSlot {
 }
 
 export interface ISiteManagedClasses {
+    site__headerTitle: string;
+    site__paneToc: string;
+    site__paneTocRow: string;
+    site__paneTocTitle: string;
     site__paneToggleButton: string;
-    site__paneTogglButtonIcon: string;
+    site__paneToggleButtonIcon: string;
+    site__paneToggleButtonIconLayout: string;
 }
 
 const styles: ComponentStyles<ISiteManagedClasses, IDevSiteDesignSystem> = {
-    site__paneToggleButton: {
-        width: "32px",
-        height: "32px",
+    site__headerTitle: {
+        verticalAlign: "middle"
+    },
+    site__paneToc: {
         padding: "0"
     },
-    site__paneTogglButtonIcon: {
-        display: "inline-block",
-        width: "32px",
-        height: "32px"
+    site__paneTocRow: {
+        display: "flex",
+        flexWrap: "nowrap",
+        alignItems: "center",
+    },
+    site__paneTocTitle: {
+        fontWeight: "bold",
+        marginLeft: "-8px",
+        textOverflow: "ellipsis",
```

Comment:
We should create a utility to apply ellipsis cross browser. If we want to do that in a new task, can we add a TODO?